build/stf-run-ci/defaults/main.yml

```diff
@@ -20,21 +20,21 @@ __internal_registry_path: image-registry.openshift-image-registry.svc:5000
 __deploy_minio_enabled: false
 __loki_skip_tls_verify: false
 __golang_image_path: quay.io/infrawatch/golang:1.16
-__loki_image_path: quay.io/infrawatch/loki:2.2.1
+__loki_image_path: quay.io/infrawatch/loki:2.4.2
 
 sgo_image_tag: latest
 sto_image_tag: latest
 sg_core_image_tag: latest
 sg_bridge_image_tag: latest
 prometheus_webhook_snmp_image_tag: latest
 loki_operator_image_tag: latest
-new_operator_sdk_version: v1.5.0
+new_operator_sdk_version: v1.11.0
```

Comment:
Thanks Chris, yea this is because Loki Operator is built with a newer version of operator-sdk than what we use for our components. This was actually one of the issues I had to deal with as part of the operator-sdk migration.

This would get dealt with should we decide to continue with the effort in migrating to operator-sdk 1.16 (or otherwise). Ideally Loki would exist somewhere that we could use. I'm thinking that instead of _building_ Loki Operator directly into our test framework, we should actually consider just making it available in a CatalogSource / index image like what is available for Prometheus, ECK etc, and maybe just pull all of this logic out entirely.

We don't maintain Loki and are unlikely to be the ones that will productize it. Keeping this kind of build logic in our test framework is, in my opinion, just making our test framework more complicated for no reason.

I might look at how hard it would be to just create the bundle and index image now that I've got some > 1.0 operator-sdk experience. I think it will be relatively straight forward.

No changes required for this PR. I might see if I can hack something together quickly and pull out some of the custom logic in our test framework to clean it up.